### DISCUSSION TITLE
Poor SSB RX compared to IJV

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#358] [General]
Side band. https://www.youtube.com/watch?v=EzuDTiOunnY

**Comment by egzumer** [comment]
It probably uses different gain settings and/or bandwidth. I have no way to test this.

**Comment by pietss** [comment]
Premise. Of course I understand that you firmware developers are acting almost like tightrope walkers to squeeze as much as possible out of this radio and this always involves compromises, some in one direction and some in another, so if I add themes it's not to criticize since I don't even have the expertise. 

I report an answer from Fabrizio (IJV) to a question https://www.youtube.com/watch?v=vto1ScDevuk&list=PLZFTnLrHyeRGiIOE8hUJFVYCMO_kc_qgQ&index=7
I hope the translation is correct enough.

“I have another identical device but with ezgumer mod, the thing I mainly like about that firmware is the possibility of being able to adjust the maximum and minimum brightness (when the radio is in stand by and the bl time expires). Could you implement it?”

“no, for a simple reason, I based the entire development of the firmware on maximum sensitivity and quality of reception, inserting a LED dimming function done with PWM would mean introducing further internally generated noise. this would be counterproductive for quality purposes. You can notice this kind of disturbance by trying to receive with the USB charging connected...”

**Reply by egzumer** [reply]
It introduces audio noise, but it is rarely noticable. You can simply turn of the dimming i.e. set it to max, it doesn't have to be used, it is up to the user.

**Comment by Hunterfnet** [comment]
overall is nice and funny having more noise scanning channels , it's a normal design ..... no need change nothing , all is perfect for receive more noise when signal level ups up and don't hear the transmission like on the main video showed here.
https://github.com/egzumer/uv-k5-firmware-custom/issues/310
on CW for now nothing for compare , i wish you all , a better and fantastic 2024

**Comment by AsciiWolf** [comment]
> Side band. https://www.youtube.com/watch?v=EzuDTiOunnY

Was anyone able to test this properly? There were some comments under that video mentioning different RX AGC settings/implementation between these firmwares as a cause of this, but I am not able to test this on my current QTH.
